caravel/assets/visualizations/big_number.js

```diff
@@ -6,6 +6,7 @@ require('./big_number.css');
 
 var px = require('../javascripts/modules/caravel.js');
 
+
```

Comment:
Thanks for the feedback! I'll make those changes. I think npm run lint is listed in contributing.md under testing, but when I ran it, I got a bunch of npm errors. Now that I look I can see the lint errors in the travis build before the npm errors, that wasn't showing up locally.